src/http/request_context.rs

```diff
@@ -11,7 +12,7 @@ use crate::config::Server;
 
 #[derive(Setters)]
 pub struct RequestContext {
-  pub data_loader: DataLoader<HttpDataLoader, HashMapCache>,
+  pub data_loader: Arc<Option<DataLoader<HttpDataLoader, NoCache>>>,
```

Comment:
Since we have only one instance of a DL, it doesn't need to be in request context. Instead it should be in server context.